client/cassh_docker.sh

```diff
@@ -7,4 +7,4 @@ docker run --rm -it \
   --volume=${HOME}/.cassh:${HOME}/.cassh:ro \
   --volume=${HOME}/.ssh:${HOME}/.ssh \
   --workdir=${HOME} \
-  nbeguier/cassh-client "$@"
+  leboncoin/cassh-client "$@"
```

Comment:
looks to be [`leboncoin/cassh`](https://hub.docker.com/r/leboncoin/cassh/)

Reply:
Yeah ... maybe we will bee `cassh` for the client and `cassh-server` for the server.